storages/redis/redis.go

```diff
@@ -109,12 +109,12 @@ func (s *Storage) resetData(chatId, userId int64) error {
 			s.resetDataBatchSize,
 		).Result()
 		if err != nil {
-			return pkg_errors.Wrap(err, "scan")
+			return wrapError(err, "scan")
```

Comment:
it duplicates the message like

fsm-telebot/storage/redis: reset data: fsm-telebot/storage/redis: test error: test error

Reply:
oh 😕, I didn't notice